### ISSUE TITLE
i have a question about the learning rate

### ISSUE THREAD

**Issue by xgbj** [ZJULearning/pixel_link#11]
When I used the script to train icdar2015 datasets, Nan appeared after roughly  20k rounds. Why is this learning rate set to 1e-3 first and then fixed to 1e-2?

**Comment by kulkarnivishal** [comment]
Any solution to this?

**Comment by northeastsquare** [comment]
try gradient clip

**Comment by alpace** [comment]
Have you solved it?

**Comment by Doan-Nguyen** [comment]
> Have you solved it?

i fixed problem, set learning_rate = 1e-3 in 100 step & learning_rate = 1e-4.
